node-notes-cli/app.js

```diff
@@ -0,0 +1,45 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function parse() {
+  const file = await readFile('data.json');
+  const parse = await JSON.parse(file);
+  return parse;
+}
+
+const data = await parse();
+
+async function notes() {
+  const list = data.notes;
+  console.log(list);
```

Comment:
This logs the surround `{}`. It would be nice to not have those in the output since "normal" users would be confused by them.